Rymera-Web-Co/wcvendors
# Fixed: Bulk Actions in WP Admin's Orders Page Causes Orders to be Mar…

…ked as Shipped

### All Submissions:






### Changes proposed in this Pull Request:



Closes #889 .
- Bail out if this is not a bulk mark shipped action.
### How to test the changes in this Pull Request:

Go to the Orders page in WP admin

1. Select one or more orders
2. Select an action other than "Mark shipped" in the Bulk Actions dropdown, e.g. Change order status to processing
3. Run the bulk action
4. Observe that the selected orders are not marked shipped for all vendors

### Other information:
